.gitignore

```diff
@@ -35,6 +35,7 @@ tools/vms-generator/vms-generator
 .coverprofile*
 coverage.html
 manifests/**/*.tmp
+cluster-up/**/*.cert
```

Comment:
Please be specific about our SR-IOV cert files. There is no precedent against having any cert files in repository.

Reply:
done